### ISSUE TITLE
Review upgrades/achievements values and rewards.

### ISSUE THREAD

**Issue by DainDwarf** [DainDwarf/IncrementalEmpire#66]
Basically, restart a game and see how that evolves. Change costs when they seem unavailable or too easy.

Also, think about adding upgrades to lock some behaviors.

Also, think about achievements you could add.

**Comment by DainDwarf** [comment]
Things to lock:
* Material (route, display, both empire and template, even capital display if that is not linked to the tribe unlock)
 -> Find a way to make material available before workers, so that you need to build a first production building using your imported material? Or is that too convoluted?
* Buildings (Separately? Full lvl at once? Storage, then workers, then production?)
* Worker assignment/production (as well as the worker display in empire navbar)
* Holy Building
* Building buildings?
* Buildings in templates
* Capital lvls
* Magic Anvil

Possible achievements:
* Reaching some number of ressources (under a given number of turns?) Not that easy with the storage limits
* Number of buildings?
* Filling up eden (as religious), and cave (as economical) for unlocking next capital lvl in templates?
* ???

Numbers/formula to review:
* Basically everything around buildings
* Upgrade costs now that ressources/production are limited by buildings
